### DISCUSSION TITLE
Cannot Close Window on stopped Codespace in VS Code

### DISCUSSION THREAD

**Discussion by philip-gai** [community/community#7916] [Codespaces]
Steps to reproduce:

1. Keep VS Code open with your Codespace connected
2. Let it go idle until the Codespace stops
3. ![image](https://user-images.githubusercontent.com/17363579/142667442-179d1900-fef8-4a68-90ca-dbee615916fd.png)
4. Select "Close Window"
5. It will not close. It will say "Disconnected, trying to reconnect"
6. ![image](https://user-images.githubusercontent.com/17363579/142667569-97653c20-fccf-474b-8630-320885495b9f.png)
7. When I Dismiss, it will go back to step 3 and just loop like that until I quit VS Code 😄

**Comment by n123456789k** [comment]
Do you loose access to loading files etc also? is this before 30 min timeout?

**Comment by asciimike** [comment]
I believe this is https://github.com/microsoft/vscode/issues/137468, which we're working on fixing. Apologies about the issues here :(
